.vale/styles/Flipt/spelling-exceptions.txt

```diff
@@ -43,8 +43,10 @@ okta
 otlp
 performant
 PGBouncer
+plugable
```

Comment:
```suggestion
pluggable
```

Reply:
i knew that looked weird https://english.stackexchange.com/questions/77373/plugable-or-pluggable